packages/workflow/src/interceptors.ts

```diff
@@ -18,6 +18,14 @@ export interface WorkflowExecuteInput {
   readonly headers: Headers;
 }
 
+/** Input for WorkflowInboundCallsInterceptor.handleUpdate and
+ * WorkflowInboundCallsInterceptor.validateUpdate */
+export interface UpdateInput {
```

Comment:
I think we can also add `readonly id: string` here since an interceptor may want to know the update ID

Reply:
Please qualify this id